### ISSUE TITLE
Pet feeder entities do not change

### ISSUE THREAD

**Issue by z-master42** [al-one/hass-xiaomi-miot#1667]
### Device model / 设备型号

mmgg.feeder.fi1

### Component version / 插件版本

0.7.18

### HA core version / HA版本

2024.5.x

### Integrated mode / 集成方式

Local (本地模式)

### The problem / 问题详情

The sensors sensor.xyz_doorstatus, sensor.xyz_outletstatus and sensor.xyz_device_fault do not change their status so that, for example, device errors or blockages are not recognised.

### Entity attributes / 实体属性

supported_features: 0
model: mmgg.feeder.fi1
lan_ip: 192.168.42.34
mac_address: EC:4D:3E:17:8B:39
entity_class: MiotSensorEntity
home_room: Westerstede Küche
exclude_miot_properties: outfood_num, outfood_id, contrycode, feddplan_string, factory_result
miot_type: urn:miot-spec-v2:device:pet-feeder:0000A06C:mmgg-fi1:1
state_property: pet_feeder.pet_food_left_level
pet_feeder.fault: 0
pet_feeder.pet_food_left_level: 0
clearnserve.cleantime: 8
keyserve.key_stat: 1
feedplanserve.feedplan_contr: 0
feedserve.outletstatus: 0
feedserve.doorstatus: 0
desiccant.desiccant_left_time: 0
indicator_light.on: 1
state_updater: lan
sub_entities: pet_feeder-2.feeding_measure-5, pet_feeder-2.fault-1, desiccant-11.desiccant_left_time-2, feedserve-4.outletstatus-8, feedserve-4.doorstatus-9, clearnserve-8.cleantime-1, indicator_light-3.on-1, keyserve-6.key_stat-1, pet_feeder-2.pet_food_out-1, desiccant-11.reset_desiccant_life-1, clearnserve-8.resetclean-1, indicator_light-3
pet_feeder.pet_food_left_level_desc: Normal

### Home Assistant Logs / 系统日志

No errors in the log

**Comment by z-master42** [comment]
In addition, these attributes are always set to 5. As far as I understand it, an attribute should reflect the amount of food dispensed.

feeding_measure: 5
feedserve.outfood_num: 5
feedserve.outfood_id: 5

**Comment by Azaelyus1** [comment]
Do we have any update on this? I am getting an error in the official Xiaomi Home app that the doors are blocked, but HA is not reporting any change in doorstatus or error entity state.

**Comment by z-master42** [comment]
Since version 7.20, `sensor.xxx_outfood_num` has been available as a separate sensor/entity, but it still only displays 5 and nothing else.

In addition, `binary_sensor.xxx_outletstatus` and `binary_sensor.xxx_doorstatus` have been newly created. The old entities no longer work.
However, there is some confusion with these, at least for me. 
I think door status means the lid at the top for refilling and outlet status is the small flap at the food outlet. But it seems to be the other way round, which makes no sense to me.

**Comment by Azaelyus** [comment]
Hi @z-master42, I can confirm the same. Anyway, it is just a naming problem, you can rename the entities as you wish.

What seems to be much bigger issue is that both fault and feeding door entities do not update their status. My feeder has clogged a few times in the past and I had no information about it.

**Comment by z-master42** [comment]
Yes, I renamed it a long time ago. And yes, only the sensor for the top cover works.

It's rubbish that the blocking error message doesn't work. I think there was a firmware update for the feeder some time ago that changed the mechanism. Previously, when it became stiff, it simply turned briefly in the opposite direction and then tried again. Now it just keeps turning at will so that it pushes the feeder up and the lid pops out of its catch. I also have to say that the dry food I have in there is actually too big for the appliance.
